data/fro/Tyssens_1967_EnfGuillaume/metadata.txt

```diff
@@ -0,0 +1,90 @@
+publicationType : "book"
+publicationTitle : "La Geste de Guillaume d’Orange dans les manuscrits cycliques"
+publicationDate : "1967"
+publicationPlaces:
+    - publicationPlace: "Paris"
+publicationSeries : "Bibliothèque de la Faculté de philosophie et lettres de l’Université de Liège"
+publicationNum: "178"
+publicationStemmaNum: ""
+publicationAuthors:
+    - publicationAuthor: "Tyssens, Madeleine"
+publicationPage : "187"
+publicationLink : ""
+workTitle : "Enfances Vivien"
+workViaf : ""
+workOrigDate : "1267-1300"
+workOrigPlace : ""
+workAuthors:
+    - workAuthor: ""
+      workAuthorViaf: ""
+workGenre : "chanson de geste"
+workLangCode : "fro"
+stemmaType : "reconstructed"
+drawnStemma : "true"
+contam : "no"
+extraStemmContam : "no"
+rootType : "original"
+completeWits : "included"
+sourceText : "NA"
+derivatives : "excluded"
+contributors:
+    - contributor: "Jean-Baptiste Camps"
+      contributorORCID: "0000-0003-0385-7037"
+note : "The witnesses included are mostly complete with the exception of the fragments and the Oxford manuscript. The details about the structure of the descendants of A are given independently p. 356 in the text; p. 367 for B1 and B2; for C and C2, p. 377. For the Enfances Vivien, the stemma is based on Nordfelt's stemma, with some adaptations given in the text, based on a reexamination of the tradition, and the discussion (in the following chapter) of Terracher's stemma of the Chevalerie Vivien. The encoding accounts for the modification given in the text."
```

Comment:
Are we talking about Enfances Vivien or Enfances Guillaume? And in the image there are three stemmata, which is this? Should we crop the image?

Reply:
Ok, I need to check both Enfances in Tyssens at the BEC. Will do it tomorrow !